urllib3/response.py

```diff
@@ -284,9 +299,10 @@ def stream(self, amt=2**16, decode_content=None):
             If True, will attempt to decode the body based on the
             'content-encoding' header.
         """
+        self._init_decoder()
         if self.chunked:
             for line in self.read_chunked(amt):
-                yield line
+                yield self._decode(line, decode_content, True)
```

Comment:
I'll be happy to fix that when I add support to read_chunked to decode the content and move the use of `_decode` into `read_chunked`.